Shared/en.lproj/Localizable.strings

```diff
@@ -98,7 +98,7 @@
 "Share" = "Share";
 
 /* No comment provided by engineer. */
-"Sharing" = "Sharing";
+"Sharing" = "Sharing…";
```

Comment:
?

Reply:
Sorry, no. I have several arguments against it:

- With just this change, we would become asymmetric to status "Hosting".
- We introduce a breaking point, where translators will stumble over.
- The ellipsis is often used in waiting dialogs, where the user has to wait on something to finish. This is not the case here. It's the status where the user has deliberately put the app in, and typically the user will put it out of that state again.

Reply:
Was just guessing here. (Hence my comment.)